spec/openapi.yml

```diff
@@ -696,8 +696,10 @@ components:
       in: query
       schema:
         type: string
-        example: dataset:food_delivery:public.delivery_7_days
-      description: The ID of the node.
+        example: dataset:food_delivery:public.delivery_7_days or job:my-namespace:my-job
```

Comment:
The example should include either a dataset node or job node, but not both:

<img width="723" alt="Screen Shot 2022-08-23 at 2 51 15 PM" src="https://user-images.githubusercontent.com/1553185/186272947-6f7ebe77-7d70-441e-9c15-a9d06ac2a1d5.png">

The `example` should be a valid query param for `nodeId`

Reply:
Is there any option of having more than one entries for example? I was looking for ways to create example for dataset and job...